pocs/mount/__init__.py

```diff
@@ -1 +1,55 @@
+from glob import glob
+
 from pocs.mount.mount import AbstractMount  # pragma: no flakes
+from pocs.utils import error
+from pocs.utils import load_module
+from pocs.utils.location import create_location_from_config
+from pocs.utils.logger import get_root_logger
+
+
+def create_mount_from_config(config, mount_info=None, earth_location=None):
```

Comment:
```suggestion
def create_mount_from_config(config, mount_info=None, earth_location=None, *args, **kwargs):
```

Not sure if I said this on the other ones, but let's pass values down to the actual mount creation. See below.